output/src/main/scala-3/fix/EitherFoldTest.scala

```diff
@@ -0,0 +1,24 @@
+package fix
+
+object EitherFoldTest {
+  def f1[A, B, C](x: Either[(A, A), B], g1: A => C, g2: B => C): C = {
+    x.fold({ case (a1, a2) => g1(a1)
+        g1(a2) }, r => g2(r))
+  }
+
+  def f2[A, B, C](x: Either[A, B], g1: A => C): Unit = {
+    x.fold(l => g1(l)
```

Comment:
```
 [error] -- [E050] Type Error: D:\a\scalafix-rules\scalafix-rules\output\src\main\scala-3\fix\EitherFoldTest.scala:10:18 
[error] 10 |    x.fold(l => g1(l)
[error]    |                ^^^^^
[error]    |           method apply in trait Function1 does not take more parameters
[error]    |
[error]    | longer explanation available when compiling with `-explain`
```

Reply:
https://github.com/scalameta/scalameta/issues/3570#issuecomment-1960436928

Reply:
This was fixed in https://github.com/scalameta/scalameta/pull/3581 (which introduced [another change](https://github.com/scalameta/scalameta/issues/3594)).